client/main.coffee

```diff
@@ -63,44 +71,71 @@ Leaderboard = ReactClass
     players: []
     selectedPlayerId: ''
   incPlayerScore: ->
-    Players.update(@selectedPlayer._id, {$inc: {score: 5}})
+    Players.update @selectedPlayer._id
+    , 
```

Comment:
I try to avoid lingering commas at all costs, I'd move it up to just after `_id`.